defs.bzl

```diff
@@ -7,17 +12,26 @@ def _pip_repository_impl(rctx):
     result = rctx.execute(
         [
             rctx.which(rctx.attr.python_interpreter),
-            rctx.path(rctx.attr._script),
+            rctx.path(rctx.attr._script).dirname,
             "--requirements",
             rctx.path(rctx.attr.requirements),
             "--repo",
             "@%s" % rctx.attr.name,
         ],
-
-        environment = rctx.attr.wheel_env,
+        environment={
+            # Manually construct the PYTHONPATH since we cannot use the toolchain here
```

Comment:
We previously hacked together the PYTHONPATH in the __init__ script. This led to some annoying import paths. By building the path here, we can leverage the `repositories.bzl` dependencies, and make the imports work nicely.